client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/OpenSearchCommand.java

```diff
@@ -1,12 +1,13 @@
 package org.evomaster.client.java.instrumentation;
 
 import java.io.Serializable;
+import java.util.List;
 
 public class OpenSearchCommand implements Serializable {
     /**
      * Name of the index that the operation was applied to
      */
-    private final Object index;
+    private final List<String> index;
```

Comment:
update docs based on new type. ie, how can a list of strings be a name?

Reply:
Updated docs following OpenSearch SearchRequest docs.
```
    /**
     * A comma-separated list of index names to search; use <code>_all</code> or
     * empty string to perform the operation on all indices
     * <p>
     * API name: {@code index}
     */
```